packages/editor/src/components/editor-help/style.scss

```diff
@@ -0,0 +1,33 @@
+.sectionContainer {
```

Comment:
Thanks for this information. I'd be interested in how you would change the names of these styles specifically so I can see an example of it applied since there don't seem to be any currently in the native styles code. Thanks!

Reply:
Sure thing! I'll put something together and share at a later time.

Reply:
Hey @AmandaRiu. 👋🏻 Following up to share an example diff of how these styles might be rewritten following the BEM naming conventions. I hope it's helpful. 

<details><summary>Help BEM Styles</summary>

```diff
diff --git a/packages/editor/src/components/editor-help/index.native.js b/packages/editor/src/components/editor-help/index.native.js
index 3a53c94922..7c3be080a2 100644
--- a/packages/editor/src/components/editor-help/index.native.js
+++ b/packages/editor/src/components/editor-help/index.native.js
@@ -34,21 +34,21 @@ const HELP_TOPICS = [
 
 function EditorHelpTopics( { isVisible, onClose } ) {
 	const bottomSheetHeaderTitleStyle = usePreferredColorSchemeStyle(
-		styles.bottomSheetHeaderTitle,
-		styles.bottomSheetHeaderTitleDark
+		styles.help__heading,
+		styles[ 'help__heading--dark' ]
 	);
 
 	return useMemo( () => (
 		<BottomSheet
 			isVisible={ isVisible }
 			onClose={ onClose }
-			contentStyle={ styles.contentContainer }
+			contentStyle={ styles.help }
 			hideHeader
 			hasNavigation
 		>
 			<BottomSheet.NavigationContainer animate main>
 				<BottomSheet.NavigationScreen name={ __( 'Topics' ) }>
-					<View style={ styles.bottomSheetHeader }>
+					<View style={ styles.help__header }>
 						<Text
 							style={ bottomSheetHeaderTitleStyle }
 							maxFontSizeMultiplier={ 3 }
@@ -56,10 +56,10 @@ function EditorHelpTopics( { isVisible, onClose } ) {
 							{ __( 'How to edit your site' ) }
 						</Text>
 					</View>
-					<View style={ styles.separator } />
+					<View style={ styles.help__separator } />
 					<PanelBody
 						title={ __( 'The basics' ) }
-						style={ styles.sectionContainer }
+						style={ styles.help__section }
 					>
 						{ /* Print out help topics */ }
 						{ HELP_TOPICS.map( ( topic ) => {
diff --git a/packages/editor/src/components/editor-help/style.scss b/packages/editor/src/components/editor-help/style.scss
index 2a0299588b..2c3be45b6e 100644
--- a/packages/editor/src/components/editor-help/style.scss
+++ b/packages/editor/src/components/editor-help/style.scss
@@ -1,22 +1,22 @@
-.contentContainer {
+.help {
 	padding: 0;
 }
 
-.sectionContainer {
+.help__section {
 	padding: 0 16px;
 }
 
-.separator {
+.help__separator {
 	background-color: $light-gray-400;
 	height: 1px;
 	width: 100%;
 }
 
-.separatorDark {
+.help__separator--dark {
 	background-color: $gray-70;
 }
 
-.bottomSheetHeader {
+.help__header {
 	flex-direction: row;
 	align-items: center;
 	padding-right: $grid-unit-20;
@@ -24,7 +24,7 @@
 	min-height: 44px;
 }
 
-.bottomSheetHeaderTitle {
+.help__heading {
 	color: $light-primary;
 	text-align: center;
 	font-weight: 600;
@@ -32,6 +32,6 @@
 	flex: 2;
 }
 
-.bottomSheetHeaderTitleDark {
+.help__heading--dark {
 	color: $dark-primary;
 }
```

</details>